src/icu/charset_detector.cr

```diff
@@ -6,54 +6,52 @@ class ICU::CharsetDetector
     getter confidence : Int32
 
     def initialize(csmatch : LibICU::UCharsetMatch)
-      ustatus = uninitialized LibICU::UErrorCode
+      @ustatus = uninitialized LibICU::UErrorCode
```

Comment:
Isn't there another way to get this variable being saved on the heap, I'm not really fan of using an attribute for that ? (using malloc maybe ?)